language/tools/move-unit-test/tests/test_sources/native_signer_creation.move

```diff
@@ -0,0 +1,51 @@
+module 0x1::M {
+    use 0x1::UnitTest;
+    use 0x1::Vector;
+    use 0x1::Signer;
+
+    struct A has key {}
+
+    fun has_a(a: address): bool {
+        exists<A>(a)
+    }
+
+    #[test_only]
+    fun setup_storage(sig: &signer) {
+        move_to(sig, A { })
+    }
+
+    #[test]
+    fun test_exists() {
+        let num_signers = 10;
+        let i = 0;
+
+        let signers: vector<signer> = UnitTest::create_signers_for_testing(num_signers);
+        while (i < num_signers) {
+            setup_storage(Vector::borrow(&signers, i));
+            i = i + 1;
+        };
+
+        i = 0;
+
```

Comment:
nit no whitespace. I tend to like them only after `}` or in big breaks in logic. Makes sense to group the `i = 0` with the loop IMO